client/src/components/FileCard.jsx

```diff
@@ -0,0 +1,121 @@
+import './FileCard.css'
+
+import { Folder, DocumentTxt, DocumentImage, Play } from 'grommet-icons'
+
+const FileCard = ({ name, path, type, size }) => {
+  const splittedNameArray = name.split('.')
+  const fileExtension = splittedNameArray[splittedNameArray.length - 1]
+
+  const renderFileIcon = (fileExtension) => {
+    if (fileExtension === 'txt') {
+      return <DocumentTxt color='green' />
+    } else if (fileExtension === 'jpg') {
+      return <DocumentImage color='yellow' />
+    } else if (fileExtension === 'mp4') {
+      return <Play color='blue' />
+    } else {
+      return <DocumentTxt color='green' />
+    }
+  }
+
+  return (
+    <li>
+      {type === 'folder' ? (
+        <div className='file__info'>
+          <Folder color='magenta' size='medium' />
+          <a href={`/api/content?path=${path}`} className='file__name'>
+            {name}
+          </a>
+        </div>
+      ) : (
+        <div className='file__info'>
+          {renderFileIcon(fileExtension)}
+          <div className='file__description--container'>
+            <a href={`/api/content?path=${path}`} className='file__name'>
+              {name}
+            </a>
+            <p className='file__subtitle'>{size}</p>
+          </div>
+        </div>
+      )}
+    </li>
+  )
+}
+
+export default FileCard
+
+// name	"@root"

```

Comment:
if you need some notes with the response that you get from server, create folder `client/src/__mocks__/` and put your response json file there, example: `files.json`